sdk/go/src/sawtooth_sdk/client/signer.go

```diff
@@ -91,6 +92,22 @@ func SHA256(input []byte) []byte {
 	return hash.Sum(nil)
 }
 
+// -- WIF --
+
+// PrivToWif converts a private key generated to a WIF string
+func PrivToWif(priv []byte) string {
+	extended := append([]byte{0x80}, priv...)
+	checksum := SHA256(SHA256(extended))[:4]
+	extcheck := append(extended, checksum...)
+	return base58.Encode(extcheck)
```

Comment:
Is this just translating to base58? You might need checkencode(). I'm not sure off the top of my head if that's sufficient for wif or if there are other requirments.